spyder/plugins/editor/widgets/base.py

```diff
@@ -170,12 +170,17 @@ def keyPressEvent(self, event):
         shift = event.modifiers() & Qt.ShiftModifier
         ctrl = event.modifiers() & Qt.ControlModifier
         modifier = shift or ctrl or alt
-        if key in (Qt.Key_Return, Qt.Key_Enter) or key == Qt.Key_Tab:
-            self.item_selected()
+        if key in (Qt.Key_Return, Qt.Key_Enter, Qt.Key_Tab):
+            # Check that what was selected can be selected,
+            # otherwise timing issues

```

Comment:
Please, create an issue for this and add here the reference to the issue

Reply:
I created an issue for each problem i saw